certified-connectors/Mailjet/readme.md

```diff
@@ -0,0 +1,3 @@
+## Authentication:
```

Comment:
@GeoffryAbiFarah Can you please refer a sample readme.md [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/custom-connectors/AzureKeyVault/Readme.md) and improve your connector's readme.md